jMolecules/attributeconverter.md

```diff
@@ -1,6 +1,14 @@
 # jMolecules Associations können nicht richtig konvertiert werden
+
+## Meta
+Status: unverständlich
+Fragen: Kann es sein, dass das Fehlen von `@ElementCollection` hier mit reinspielt?
```

Comment:
Das Problem war, dass, wenn ich mehrere `AttributeConverter` für `Association` hatte, der falsche gewählt wurde. In der [jMolecules Spring Integration Doc](https://github.com/xmolecules/jmolecules-integrations/tree/main/jmolecules-spring#base-class-for-jpa-attributeconverter-implementations-to-persist-associations) wird der `AssociationAttributeConverter` beschrieben. Als ich meine `AttributeConverter` implementiert habe, hatte ich diesen Teil der Doc noch nicht gefunden. Daher habe ich die JPA `AttributeConverter` genutzt. Diese konnten jedoch nicht mit Generics umgehen und es wurde irgendein `AttributeConverter` für die `Association` genutzt. Den Teil der Doc hatte ich erst gefunden, nachdem, ich meinen Workaround implementiert hatte.